povm_toolbox/quantum_info/multi_qubit_povm.py

```diff
@@ -72,3 +74,27 @@ def get_prob(
         if not isinstance(rho, SparsePauliOp):
             rho = Operator(rho)
         return self.analysis(rho, outcome_idx)
+
+    def draw_bloch(
+        self,
+        *,
+        title: str = "",
+        figure: Figure | None = None,
+        axes: Axes | list[Axes] | None = None,
+        figsize: tuple[float, float] | None = None,
+        font_size: float | None = None,
+        colorbar: bool = False,
+    ) -> Figure:
+        """Draw the Bloch vectors of a :class:`.MultiQubitPOVM` instance.
+
+        Args:
+            title: A string that represents the plot title.
+            fig: User supplied Matplotlib Figure instance for plotting Bloch sphere.
+            ax: User supplied Matplotlib axes to render the bloch sphere.
```

Comment:
```suggestion
            figure: User supplied Matplotlib Figure instance for plotting Bloch sphere.
            axes: User supplied Matplotlib axes to render the bloch sphere.
```
Possibly the same elsewhere. Please also ensure that the order is correct and consistent everywhere (it is here)